tests/test_samples.py

```diff
@@ -63,6 +69,14 @@ def test_read_getdist():
     mcmc.plot_1d(['x0', 'x1', 'x2', 'x3'])
 
 
+@pytest.mark.skipif('montepython' not in sys.modules,
+                    reason="requires montepython package")
```

Comment:
In an ideal world, for users the montepython reader would raise the appropriate exception if users try to read montepython chains when montepython is not installed. Can you see how this might be implemented? Then your test could actually test for this behaviour by checking that the exception is raised when it's not installed, and that the code completes when it is.

Reply:
Good point, changed in commit [79933a6](https://github.com/williamjameshandley/anesthetic/pull/14/commits/79933a66d9fbb5f510b344f5caff11c790a808a0) to work now with `xfail` and not with `skipif`.